eos_data_distribution/avahi_monitor.py

```diff
@@ -188,6 +192,12 @@ def remove_nexthop(self, faceUri):
         return check_call(["nfdc", "remove-nexthop",
                            str(SUBSCRIPTIONS_SOMA), faceUri])
 
+    def start_edge(self):
+        return check_call(["systemctl", "start", "edd-soma-subscriptions-producer"])
```

Comment:
when it advertises is currently 'is it run or not'. i'd rather have edd-ssp do one thing (and do it well) and run it when i need it from the policy part.